include/openthread/types.h

```diff
@@ -683,6 +683,8 @@ enum
     OT_CHANGED_THREAD_CHILD_REMOVED         = 1 << 11,  ///< Child was removed
     OT_CHANGED_IP6_MULTICAST_SUBSRCRIBED    = 1 << 12,  ///< Subscribed to a IPv6 multicast address
     OT_CHANGED_IP6_MULTICAST_UNSUBSRCRIBED  = 1 << 13,  ///< Unsubscribed from a IPv6 multicast address
+    OT_CPU_POWER_HI                         = 1 << 14,  ///< Hi CPU power is needed
```

Comment:
I would prefer what @jwhui proposed with more generic flags name: `OT_CHANGED_COMMISSIONER_STATE `and `OT_CHANGED_JOINER_STATE` . Then we may want to remove public `otPlatCommissioningClkChange `API and perform this clock frequency change operation only internally (in this case in DA15000 port).